etcdutl/etcdutl/backup_command.go

```diff
@@ -107,6 +107,11 @@ func newDesiredCluster() desiredCluster {
 func HandleBackup(withV3 bool, srcDir string, destDir string, srcWAL string, destWAL string) error {
 	lg := GetLogger()
 
+	if !withV3 {
+		lg.Warn("-with-v3=false is not supported since etcd v3.6. Please use v3.5.x client as the last supporting this deprecated functionality.")
```

Comment:
Please mention this if flag description.

Reply:
Thanks & Resolved.